package com.core_java.exceptionhandling;

public class FinallyBlock {

	public static void main(String[] args) {

		try {
			
		} catch (Exception e) {
			// TODO: handle exception
		}
		finally {
			System.out.println("Get Lost finally blocj=k");
		}
	}

}
